crates/turborepo-globwalk/src/lib.rs

```diff
@@ -463,6 +318,16 @@ mod test {
     #[cfg(windows)]
     const GLOB_ROOT: &str = "C\\:/"; // in globs, expect an escaped ':' token
 
+    #[test_case("a", "a" ; "no change")]
+    #[test_case("**/**", "**")]
+    #[test_case("**/**/**", "**" ; "Triple doublestar")]
```

Comment:
just curious, what does triple doublestar do? I thought `**/**` means full depth, not literally 2 deep?